Dockerfile.dev

```diff
@@ -2,7 +2,7 @@ FROM ruby:2.5.8
 
 RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
 RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-RUN apt-get update -qq && apt-get remove cmdtest && apt-get install -y nodejs postgresql-client yarn
```

Comment:
`cmdtest` is not present on the image so we don't need to remove it

Reply:
But we do nodejs postgresql-client and yarn...